browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -406,18 +407,44 @@ void BraveBrowserView::UpdateSecondaryContentsWebViewVisibility() {
     }
 
     secondary_contents_web_view_->SetVisible(true);
+    UpdateSecondaryDevtoolsLayoutAndVisibility(contents);
+
     auto* contents_layout_manager = static_cast<BraveContentsLayoutManager*>(
         contents_container()->GetLayoutManager());
     contents_layout_manager->show_main_web_contents_at_tail(
         second_tile_is_active_web_contents);
   } else {
     secondary_contents_web_view_->SetWebContents(nullptr);
     secondary_contents_web_view_->SetVisible(false);
+    secondary_devtools_web_view_->SetVisible(false);
```

Comment:
don't need this here? `secondary_devtools_web_view_->SetWebContents(nullptr);`

Reply:
That'd be better.